projects/mci/src/mci_output.c

```diff
@@ -23,6 +25,12 @@ static float s_velocity_lookup[] = {
 
 static float s_max_regen_in_throttle = 0.0f;
 static float s_regen_threshold = 0.0f;
+static float s_actual_velocity_ms = 0.0f;
+
+// Updateds currents_current_velocity_ms
```

Comment:
Typo: `Updateds -> Updates`, and I think the variable is s_actual_velocity_ms lol

Reply:
done